LandSandBoat/server
# [Magic] Implement Innin Ninjutsu damage multiplier

**_I affirm:_**

## What does this pull request do?

- Implements damage multiplier from Innin
- Applies "Koga Hatsuburi" type gear bonuses to magic damage.
- Adds modifier to Innin effect.
- Makes Innin effect decay a little better, making it respect minimum values.
- Found **A** data source for Ninjutsu spells, so updated parameters to refect them as opposed to using made-up data.
- Corrected Ninjutsu skill multiplier not using maximum skill level dependant on spell tier.

## Steps to test these changes
Review by commit.

Use innin.
Use a ninjutsu spell facing a mob. See damage.
Now do it from his back. See damage be greater.
